flink-libraries/flink-table-common/src/main/java/org/apache/flink/table/descriptors/DescriptorProperties.java

```diff
@@ -1134,7 +1134,7 @@ public void validateArray(String key, Consumer<String> elementValidation, int mi
 		}
 
 		// validate array elements
-		for (int i = 0; i < maxIndex; i++) {
```

Comment:
@xccui Thanks for fixing the bug. line 941 in this file also contains the bound problem. Would be great if you can also fix it. 

Best, Hequn

Reply:
Well, maybe I should create a JIRA issue for these problems. Will close this PR.
Thanks for looking into this, @hequn8128!

Best, Xingcan